src/BWMesh.hpp

```diff
@@ -1,53 +0,0 @@
-//
-//  BWMesh.hpp
-//  Delauney Mesh Generation
-//
-//  Created by Ben Rider on 1/15/25.
-//
-
-#ifndef BWMesh_hpp
-#define BWMesh_hpp
-
-#include <stdio.h>
-#include <vector>
-#include <iostream>
-#include "Point2D.hpp"
-#include "Triangle.hpp"
-
-class BWMesh {
-    
-public:
-    BWMesh();
-    std::vector<Triangle> connecArray;//connectivity array
```

Comment:
should make member functions `private:`

typically the class could be organized this way depending on what sections you needed 
`publc:`
`protected:`
`private:`

Reply:
i would also evaluate which methods really need exposed from this class and which are internal details